src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/DigitalCatalogSchema.cs

```diff
@@ -162,7 +163,48 @@ public void Build(ISchema schema)
                 return await ResolveCategoriesConnectionAsync(_mediator, context);
             });
 
-            schema.Query.AddField(categoriesConnectionBuilder.FieldType);
+            schema.Query.AddField(categoriesConnectionBuilder.FieldType);
+
+            var propertiesConnectionBuilder = GraphTypeExtenstionHelper.CreateConnection<PropertyType, object>()

```

Comment:
I recommend using `filter` parameter for filter properties by extended syntax,  rather than use hardcoded set of possible properties. See example https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Schemas/OrderSchema.cs#L94

How to map filter expression to criteria https://github.com/VirtoCommerce/vc-module-experience-api/blob/dev/src/XPurchase/VirtoCommerce.ExperienceApiModule.XOrder/Queries/SearchOrderQueryHandler.cs#L26